lms/djangoapps/verify_student/apps.py

```diff
@@ -18,3 +18,4 @@ def ready(self):
         Connect signal handlers.
         """
         from . import signals  # pylint: disable=unused-variable
+        from . import tasks    # pylint: disable=unused-variable
```

Comment:
Why is this needed?

Reply:
When the app initializes, I would love to import all my tasks so that they are auto-discovered by celery.